inst/subtype.model.R

```diff
@@ -1,6 +1,5 @@
-path.subtype.model <- '/hot/project/disease/ProstateTumor/PRAD-000101-MethySubtypes/data/2024-11-13_example-data_for-predicting-subtypes.Rdata';
-
-load(path.subtype.model);
+source('config.R') # see project PRAD-000101-MethySubtypes/PrCaMethy/config.R
```

Comment:
This `source('config.R');` solution should be fine for scripts using the package.